inst/extdata/bic_animal_tissue_code.tsv

```diff
@@ -21,7 +21,7 @@ T19	Human Mycoplasma	T19 Human Mycoplasma	t19-mycoplasma
 T20	Human Muscle Histology	T20 Human Muscle Histology	t20-muscle-histology
 T21	Human Adipose Histology	T21 Human Adipose Histology	t21-adipose-histology
 T30	Blood RNA	T30 Rat PaxGene RNA	t30-blood-rna
-T31	EDTA Plasma	T31 Rat Plasma	t31-plasma

```

Comment:
why remove this tissue code? isn't it still used in the PASS data releases?

Reply:
oh, nice catch. I downloaded a copy from where we keep the source of truth, but for whatever reason that was missed there. I put it back. Thanks!